sys/kern/tmpfs.c

```diff
@@ -965,7 +965,7 @@ static int tmpfs_mount(mount_t *mp) {
   /* Allocate the root node. */
   vattr_t va;
   vattr_null(&va);
-  va.va_mode = S_IFDIR | ACCESSPERMS;
+  va.va_mode = S_IFDIR | ACCESSPERMS | S_ISTXT;
```

Comment:
Since `tmpfs` in theory could be mounted more than once, not every instance will be mounted on `/tmp`. Hence please do not add this attribute here, but rather `do_chmod` in `mount_fs`.